extra/dist/tarantoolctl.in

```diff
@@ -25,6 +25,17 @@ int getppid(void);
 
 local TIMEOUT_INFINITY = 100 * 365 * 86400
 
+-- Use getenv_safe when it's present. When not, fall back to plain getenv.
+-- getenv_safe accepts name and size, getenv accepts only name. So we can call
+-- both functions with same arguments. Plain getenv will simply ignore the extra
+-- argument.
+local getenv = os.getenv_safe or os.getenv
+-- How many digits there are in a 8 byte integer + 1 digit for null terminator.
+-- UIDs and GIDs actually use 4 byte integers, but better be on the safe side.
+local UINT64_STRSIZE_MAX = 21
+local PATH_MAX = os.PATH_MAX or 1024
+local USER_SIZE_MAX = 257
```

Comment:
257? Is +1 for nul terminator? Lua knows nothing about nul terminator. Passing the max length for the second argument to `getenv_safe` would look more natural IMO. And we could do the same in the C version.

Anyway, a user name in Tarantool is limited by `BOX_NAME_MAX`, which equals 65000.

Reply:
I see, sorry for misunderstanding.

Still, 257 looks unnatural. For a Lua developer 256 would look better IMO. I'm not 100% sure though.

Reply:
> But for C passing buffer size would be more consistent IMO. `snprintf` operates with buffer size, not string length, for example. Also `PATH_MAX` defines buffer size, not max path len.

For a C developer, max string length would look okay IMO. For example, `strncpy` takes max length, not buffer size.

Reply:
Anyway, now `os.getenv()` is overloaded and there is no more `os.getenv_safe`.